modules/cli/cmds/cleanup.go

```diff
@@ -0,0 +1,79 @@
+package cmds
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/rancher/fleet/modules/cli/cleanup"
+	command "github.com/rancher/wrangler-cli"
+)
+
+func NewCleanUp() *cobra.Command {
+	cmd := command.Command(&CleanUp{}, cobra.Command{
+		Use:   "cleanup [flags]",
+		Short: "Clean up outdated cluster registrations",
+	})
+	command.AddDebug(cmd, &Debug)
+	return cmd
+}
+
+type CleanUp struct {
+	Min    string `usage:"Minimum delay between deletes (default: 10ms)" name:"min"`
```

Comment:
Just curious: why are we not using ints as done [here](https://github.com/rancher/fleet/pull/1689/files#diff-293e8fff71461cfffd5ae98b564861ced2d854fefaf7c8dc761ce32ff6ec915dR20)?
Wouldn't that prevent the below checks on arguments being parseable as durations?
Or is the plan to include such checks (including the one on `max < min`) in `master`?

Reply:
Ah, because I backported from this PR https://github.com/rancher/fleet/pull/1697 .. which I now see wasn't merged yet.